### PR TITLE
Service Bus : fixed the typo in type of NetworkRuleSets

### PR DESCRIPTION
**Pull request by v-Ajnava**
fixed the typo in type of Microsoft.ServiceBus/Namespaces/NetworkRuleSets

### CODE REVIEW

File: schemas/2017-04-01/Microsoft.ServiceBus.json [hunk 1]
```diff
@@ -702,6 +702,40 @@
       ],
       "description": "Microsoft.ServiceBus/namespaces/AuthorizationRules"
     },
+    "namespaces_NetworkRuleSets_childResource": {
```
Comment (TianoMS) [inline schemas/2017-04-01/Microsoft.ServiceBus.json]: >namespaces_NetworkRuleSets_childResource [](start = 5, length = 40)

this is not used anywhere.

File: schemas/2017-04-01/Microsoft.ServiceBus.json [hunk 2]
```diff
@@ -702,6 +702,40 @@
...
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.ServiceBus/Namespaces/NetworkRuleSets"
```
Comment (TianoMS) [inline schemas/2017-04-01/Microsoft.ServiceBus.json]: >Microsoft.ServiceBus/Namespaces/NetworkRuleSets [](start = 13, length = 47)

for child resource, the type should be the short one - "NetworkRuleSets".

File: schemas/2017-04-01/Microsoft.ServiceBus.json (hunk 1 above)
Comment (v-Ajnava) [reply schemas/2017-04-01/Microsoft.ServiceBus.json]: Resolved

File: schemas/2017-04-01/Microsoft.ServiceBus.json (hunk 2 above)
Comment (v-Ajnava) [reply schemas/2017-04-01/Microsoft.ServiceBus.json]: resolved

**Pull request merged**